app/src/main/java/chat/rocket/android/push/PushConstants.java

```diff
@@ -0,0 +1,73 @@
+package chat.rocket.android.push;
+
+public interface PushConstants {
+  String COM_ADOBE_PHONEGAP_PUSH = "com.adobe.phonegap.push";
+  String REGISTRATION_ID = "registrationId";
+  String FOREGROUND = "foreground";
+  String TITLE = "title";
+  String NOT_ID = "notId";
```

Comment:
While I did add to the constants, I did not chance the existing ones. That's a class from the Cordova plugin and I would like to keep changes at a little.

I'll move my changes to the end fo the file and flag as so.